test/e2e/webdriver/chrome.js

```diff
@@ -110,6 +110,7 @@ class ChromeDriver {
     return {
       driver,
       extensionUrl: `chrome-extension://${extensionId}`,
+      extensionId,
```

Comment:
Extracted to a separate PR: https://github.com/MetaMask/metamask-extension/pull/30444